common/metrics/defs.go

```diff
@@ -60,9 +60,9 @@ type (
 
 // MetricUnit supported values
 const (
-	Dimensionless = "dimensionless"
-	Milliseconds  = "milliseconds"
-	Bytes         = "bytes"
+	Dimensionless = "1"
```

Comment:
Would be good to mention we try to use same value as https://pkg.go.dev/golang.org/x/exp/event#Unit